smoke-tests/ember-test-app/pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+# this tells pnpm we are a separate project with a separate install, not part of
+# the monorepo's workspaces.
```

Comment:
Adding pnpm-workspace.yaml at the root required also adding this, in order to keep the old behavior where this subdir gets an entirely independent `pnpm install`.

Arguably it would be better to just make this subdir another workspace, but that turns out to be non-trivial given NPM dependency hell, so I didn't try to attempt that change in this PR.